packages/formdescriptioneditors/frontend/sirius-components-formdescriptioneditors/src/__tests__/Group.test.tsx

```diff
@@ -49,12 +47,12 @@ const addGroupVariables: GQLAddGroupMutationVariables = {
     index: 0,
   },
 };
-const addGroupSuccessPayload: GQLAddGroupSuccessPayload = {
-  __typename: 'AddGroupSuccessPayload',
+const SuccessPayloadAddGroup: GQLSuccessPayload = {
```

Comment:
This could be a single `successPayload` value, reused in all 4 cases.

```
const successPayload: GQLSuccessPayload = {
  __typename: 'SuccessPayload',
  id: '48be95fc-3422-45d3-b1f9-d590e847e9e1',
};
```

The name `SuccessPayloadAddGroup` is confusing (to me at least) as it starts with an uppercase letter as if it was a type name.